source/bayernfahrplan/fahrplanparser/json/jsonUtils.d

```diff
@@ -0,0 +1,222 @@
+module bayernfahrplan.fahrplanparser.json.jsonutils;
+
+import std.json : JSONValue;
+import std.datetime : DateTime, Date, TimeOfDay;
+import std.conv : to;
+import fluent.asserts : should;
+
+public:
+// Todo: Implement proper error handling
+string getLine(ref JSONValue departureInfo)
```

Comment:
`departureInfo` can be `const`. Why `ref`?

Reply:
See above: departureArray can be huge and is a struct.
I will add const.